CasperSleep/nightshade
# Update node version

Bumps node version to 5.10.0, the same as Casper Rails.

**To test:** 
- Remove `node_modules/`.
- `nvm use`. You should get a message that you're using node 5.10.0.
- `npm install`
- Run nightshade. Check that everything compiles / serves.
